### PR TITLE
Move opencensus binary propagation to contrib

### PR DESCRIPTION
**Pull request by dashpole**
Issue: https://github.com/open-telemetry/opentelemetry-go/issues/93

This was originally added to opentelemetry-go in https://github.com/open-telemetry/opentelemetry-go/pull/1334, but we decided it should be moved to contrib.

This copies the binary propagation, and adds a new go module so that opencensus dependencies aren't added to users of other propagation formats.

### CODE REVIEW

File: propagators/opencensus_binary/go.mod [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module go.opentelemetry.io/otel/bridge/opencensus/binary
```
Comment (MrAlias) [inline propagators/opencensus_binary/go.mod:1]: Needs to be updated.

File: propagators/opencensus_binary/binary.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
+// Copyright The OpenTelemetry Authors
```
Comment (MrAlias) [inline propagators/opencensus_binary/binary.go:1]: I would prefer to name the directory just `opencensus` and update the package name to match. That way the use of this propagator will be `opencensus.Binary`.

File: propagators/opencensus_binary/go.mod (hunk 1 above)
Comment (dashpole) [reply propagators/opencensus_binary/go.mod]: done

File: propagators/opencensus_binary/binary.go (hunk 2 above)
Comment (dashpole) [reply propagators/opencensus_binary/binary.go]: done

File: propagators/opencensus_binary/go.mod (hunk 1 above)
Comment (dashpole) [reply propagators/opencensus_binary/go.mod]: done

**Review by MrAlias — APPROVED** [review]

**Review by Aneurysm9 — APPROVED** [review]

**Review by XSAM — APPROVED** [review]

**Pull request merged**